docs/Makefile

```diff
@@ -66,12 +66,16 @@ help:
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 	@echo "  dummy      to check syntax errors of document sources"
 
+
+gandiva_func_docs:
+	python make_gandiva_func_index.py
```

Comment:
It would probably be better to make a sphinx extension out of this.
For example in Turbogears, to generate the `cogbin` (this page https://turbogears.org/cogbin.html ) we made a `cogbin.rst` ( https://raw.githubusercontent.com/TurboGears/tgwebsite/master/src/cogbin.rst ) which contains the `.. cogbin` directive. That directive is custom and will be replaced with the list of python packages that you see in the final page. Support for the custom cogbin directive is shipped within the sphinx source itself as `cogbin.py` ( https://github.com/TurboGears/tgwebsite/blob/master/src/cogbin.py#L204-L217 ) and it's just enalbed in the doc `conf.py` ( https://github.com/TurboGears/tgwebsite/blob/master/src/conf.py#L28 )

Likewise for the YouTube directive to embedd YT videos ( https://github.com/TurboGears/tgwebsite/blob/master/src/youtube.py )

Obviously the custom directives can emit other `rst` (which I think it's what we want here) or directly `html` etc...